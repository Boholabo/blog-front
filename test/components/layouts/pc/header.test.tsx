import React from 'react';
import { render } from '../../../testUtils';
import Header from '../../../../components/layouts/pc/header';

describe('layouts/pc/header component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Header />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
