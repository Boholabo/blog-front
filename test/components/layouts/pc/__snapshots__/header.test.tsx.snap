// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layouts/pc/header component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
    id="header"
  >
    <div
      class="header-container container"
    >
      <div
        class="header-header"
      >
        <img
          alt="alt"
          class="header-logo"
        />
        <a
          class="header-link"
          href="/"
          title="$i18n.text.slogan"
        />
      </div>
      <div
        class="header-player"
      >
        <div
          class="panel"
        >
          <button
            class="prev-song button"
          >
            <i
              class="iconfont icon-music-prev"
            />
          </button>
          <button
            class="toggle-play button"
          />
          <button
            class="next-song button"
          >
            <i
              class="iconfont icon-music-next"
            />
          </button>
          <button
            class="muted-toggle button"
          >
            <i
              class="iconfont"
            />
          </button>
        </div>
        <div
          class="song"
        >
          ç©º
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
